### PR TITLE
fix: correct ecash testnet address prefix Issue: BG-3163

### PR DESCRIPTION
**Pull request by BytesOfMan**
## Description

The utxo-lib library currently lists ecash testnet addresses in the wrong format. Prefix is `ectest:`, not `ecashtest:`. This diff updates references in the utxo-lib codebase and test fixtures.

## Issue Number

External Users - #3163 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

`yarn unit-test` and all tests pass

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code compiles correctly for both Node and Browser environments
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and I have properly described any BREAKING CHANGES
- [x] The ticket or github issue was included in the commit message as a reference
- [x] I have made corresponding changes to the documentation and on any new/updated functions and/or methods - [jsdoc](https://jsdoc.app/)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: modules/utxo-lib/src/bitgo/bitcoincash/address.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function getPrefix(network: Network): string {
     case networks.ecash:
       return 'ecash';
     case networks.ecashTest:
-      return 'ecashtest';
+      return 'ectest';
```
Comment (mmcshinsky-bitgo) [inline modules/utxo-lib/src/bitgo/bitcoincash/address.ts:50]: Expected statically returned data changed. Needs breaking change in commit footer.

**Review by mmcshinsky-bitgo — DISMISSED** [review]

File: modules/utxo-lib/src/bitgo/bitcoincash/address.ts (hunk 1 above)
Comment (BytesOfMan) [reply modules/utxo-lib/src/bitgo/bitcoincash/address.ts:50]: Commit msg amended + repushed

**Comment by BytesOfMan** [comment]
Like #3160, this commit needs to be cherry picked by a Bitgo engineer for the unit tests to run correctly.

From [DEVELOPERS.md](https://github.com/BitGo/BitGoJS/blob/master/DEVELOPERS.md) at the top level of this repo:

```
Once your code is passing unit tests and has an initial review, a BitGo engineer will re-create your PR and sign your commits, and link back to the original PR.
```

**Review by OttoAllmendinger — DISMISSED** [review]
@BytesOfMan you can update the fixtures in `utxo-bin` the same way you update other fixtures: delete the files and re-run the tests

**Comment by BytesOfMan** [comment]
> can update the fixtures in utxo-bin the same way you update other fixtures: delete the files and re-run the tests

done, let me know any other hangers on.

**Comment by BytesOfMan** [comment]
when is the next version of https://www.npmjs.com/package/@bitgo/utxo-lib expected to be published?

Coming up to a couple of months now, looking to use this to implement the ecash tx support

:pray:

**Comment by OttoAllmendinger** [comment]
@BytesOfMan check out https://www.npmjs.com/package/@bitgo-beta/utxo-lib?activeTab=versions

**Comment by BytesOfMan** [comment]
unit tests will always fail in this PR since it is from an external engineer. They pass locally.

Like #3160, this commit needs to be cherry picked by a Bitgo engineer for the unit tests to run correctly.

From [DEVELOPERS.md](https://github.com/BitGo/BitGoJS/blob/master/DEVELOPERS.md) at the top level of this repo:

```
Once your code is passing unit tests and has an initial review, a BitGo engineer will re-create your PR and sign your commits, and link back to the original PR.
```

**Comment by OttoAllmendinger** [comment]
> > 
> 
> last published 6 months ago

image

**Comment by BytesOfMan** [comment]
> > > 
> > 
> > 
> > last published 6 months ago
> 
> image

tried a few of these. they instantly fail when trying to use as a drop in replacement for 7.7.0, citing missing dependencies, other things. Do you know if any of these versions are meant to be the latest master?

**Comment by OttoAllmendinger** [comment]
Opened https://github.com/BitGo/BitGoJS/pull/3472

**Comment by BytesOfMan** [comment]
rebase to latest master

**Pull request merged**